numba/cuda/tests/cudapy/test_compiler.py

```diff
@@ -130,8 +143,8 @@ def test_compile_ptx_for_current_device(self):
         def add(x, y):
             return x + y
 
-        args = (float32, float32)
-        ptx, resty = compile_ptx_for_current_device(add, args, device=True)
+        sig = (float32, float32)
+        ptx, resty = compile_ptx_for_current_device(add, sig, device=True)
```

Comment:
Similar to the comment about the tuple of args above:
```suggestion
        args = (float32, float32)
        ptx, resty = compile_ptx_for_current_device(add, args, device=True)
```